app/CommandHandling/CommandBusServiceProvider.php

```diff
@@ -0,0 +1,269 @@
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Event\CommandHandlers\UpdateUiTPASPricesHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Event\CommandHandlers\CopyEventHandler::class));
+
+                        // Organizer command handlers
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\DeleteOrganizerHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\AddLabelHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\RemoveLabelHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\ImportLabelsHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateTitleHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateDescriptionHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\DeleteDescriptionHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateAddressHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\RemoveAddressHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateWebsiteHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateContactPointHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\AddImageHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateMainImageHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\UpdateImageHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\RemoveImageHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\ImportImagesHandler::class));
+                        $commandBus->subscribe($container->get(\CultuurNet\UDB3\Organizer\CommandHandler\ChangeOwnerHandler::class));
+
+                        $commandBus->subscribe($container->get(LabelServiceProvider::COMMAND_HANDLER));
+                    }
+                );
+
+                return $commandBus;
+            }
+        );
+
+        $container->addShared(
+            'event_command_validator',
```

Comment:
Should this `event_command_validator` be exposed for reuse?

Reply:
No it can probably be moved inline indeed. But I discussed with @JonasVHG and @steftrenson to not focus on that too much now, since it's often easier to review these big refactors if the list of services stays the same. But in this PR we already did some bigger changes anyway so I'll move it inline in this case